### ISSUE TITLE
Module Filters don't working

### ISSUE THREAD

**Issue by yasarts** [projectknife/pkg-projectknife#35]
The module Filters don't working (on a protostar template).
J! 3.7.2.
Projectknife 5.0.0 Beta5

**Comment by eaxs** [comment]
Can you be more specific about how they do not work? If you select an option from one of the filters, does the page reload? Does the selected option stick after tha page has reloaded? Or does nothing happen at all?

Since the demo is running on the protostar template, where the filters are working:
https://projectknife.net/demo it is probably a javascript conflict with another extension.

**Comment by frobledoruiz** [comment]
Yes, I see filters on demo are working properly. On my site (Joomla! 3.6.5), task filters are not. (I've used only task filters so far).

Checking filters, some are working but some are not.

Working:
-Select Project
-Select Milestone
-Select Assignee
-Selec Progress
-Select Author
-Slect Access

Not working:
-Select Priority
-Select Tag

In addition to this, I see this on the frontend.
![image](https://user-images.githubusercontent.com/29613492/27709895-6d36c6f8-5ce3-11e7-893d-da8e01cf49da.png)

**Comment by yasarts** [comment]
Hi,
If you select an option from one of the filters, does the page reload? No
Does the selected option stick after tha page has reloaded? No
Or does nothing happen at all? No

And, like for frobledoruiz the display on frontend (on protostar) is not fine.

PS : I purchased the all component. The gantt part is symply incredible ! Thx a lot :-)

**Comment by eaxs** [comment]
Thank you for supporting us!
Can you email a link to your website to support@projecctknife.net? I'll have to take direct look at your site in order to help. Thanks!

**Comment by eaxs** [comment]
> Not working:
-Select Priority
-Select Tag

Thanks, I'll take a closer look.

> In addition to this, I see this on the frontend.

Yes, that's an issue with the template. We had it on the demo site too. You'll have to change the width of the select lists using CSS:
`#aside .chzn-container {
    width: 180px;
}`
